.github/workflows/build_wheels.yml

```diff
@@ -61,20 +66,20 @@ jobs:
     needs: [build_wheels, build_sdist]
```

Comment:
Please add a condition to the `publish_wheels` job so that it only runs on a tagged commit. That way, we can safely use the "manual dispatch" trigger to do a trigger the build and diagnose any issues.

I would suggest:
```yaml
    if: github.event_name == 'release' && github.event.action == 'created'
```